ansible/roles/openshift_sso_app_zabbix/tasks/main.yml

```diff
@@ -1,19 +1,19 @@
 ---
 - name: "item: openshift.master.cluster.synthetic.exists"
   zbx_host:
-    zbx_server: "{{ ozcs_zbx_server }}"
-    zbx_user: "{{ ozcs_zbx_user }}"
-    zbx_password: "{{ ozcs_zbx_password }}"
+    zbx_server: "{{ ossoz_zbx_server }}"
+    zbx_user: "{{ ossoz_zbx_user }}"
+    zbx_password: "{{ ossoz_zbx_password }}"
     state: list
     hostname: "{{ oss_clusterid }}-synthetic"
   register: out
 
```

Comment:
this zbx_host should also have a run_once: true on it. doesn't hurt anything if it doesn't have it, but runs a little faster and quieter with it.